src/widgets/histogram/content-view.js

```diff
@@ -36,20 +36,28 @@ module.exports = cdb.core.View.extend({
     this._initTitleView();
 
     var dropdown = new DropdownView({
+      normalized: this.model.get('normalized'),
```

Comment:
I'd pass the current model and change the values in the dropdown view. With that we wouldn't need another local model and passing the event to the parent view in order to change de "attribute" value.